src/components/victory-area/victory-area.js

```diff
@@ -376,12 +390,16 @@ export default class VictoryArea extends React.Component {
       "auto",
       "100%"
     );
-    const group = <g style={style.parent}>{this.renderArea(this.props)}</g>;
+    const group = <g role="presentation" style={style.parent}>{this.renderArea(this.props)}</g>;
     return this.props.standalone ?
       <svg
         style={style.parent}
         viewBox={`0 0 ${this.props.width} ${this.props.height}`}
+        role="img"
+        aria-labelledby="title desc"
       >
+        <title  id="title">{this.props.title}</title>
```

Comment:
I think these `id`s should be unique. If there are multiple charts on the page, the `aria-labelledby="title desc"` might read the wrong `id="title"` and `id="desc"`. 

It is possible the `aria-labelledby` will find the closest ones, or look at its children first, in which case this might be okay, but ...it will be safest if they are unique.